MediaBrowser.Providers/Plugins/Tmdb/TmdbExternalUrlProvider.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+
+namespace MediaBrowser.Providers.Plugins.Tmdb;
+
+/// <summary>
+/// External URLs for TMDb.
+/// </summary>
+public class TmdbExternalUrlProvider : IExternalUrlProvider
+{
+    /// <inheritdoc/>
+    public string Name => "TMDB";
+
+    /// <inheritdoc/>
+    public IEnumerable<string> GetExternalUrls(BaseItem item)
+    {
+        switch (item)
+        {
+            case Series:
+                var externalId = item.GetProviderId(MetadataProvider.Tmdb);
+                if (!string.IsNullOrEmpty(externalId))
+                {
+                    yield return TmdbUtils.BaseTmdbUrl + $"tv/{externalId}";
+                }
+
+                break;
+            case Season season:
+                var seriesExternalId = season.Series.GetProviderId(MetadataProvider.Tmdb);
+                if (!string.IsNullOrEmpty(seriesExternalId))
+                {
+                    yield return TmdbUtils.BaseTmdbUrl + $"tv/{seriesExternalId}/season/{season.IndexNumber}";
+                }
```

Comment:
Maybe restrict this to default order?
Also, for alternate orders, maybe we can use episode groups provided from TMDB?